examples/shader/gpu_readback.rs

```diff
@@ -94,7 +94,7 @@ impl Plugin for GpuReadbackPlugin {
 #[derive(Resource)]
 struct Buffers {
     // The buffer that will be used by the compute shader
-    gpu_buffer: Buffer,
+    gpu_buffer: BufferVec<u32>,
```

Comment:
Can you add a brief comment explaining to the new/unfamiliar user why you are demonstrating a wrapper type?